tests/test_http_request.py

```diff
@@ -1217,16 +1217,23 @@ def test_from_response_css(self):
                           response, formcss="input[name='abc']")
 
     def test_from_response_valid_form_methods(self):
-        body = """<form action="post.php" method="%s">
-            <input type="hidden" name="one" value="1">
-            </form>"""
```

Comment:
I think in this case the point is to avoid repeating the string, so I would switch to `.format()` instead of f-string.

Reply:
@Gallaecio,  `test_from_response_valid_form_methods` updated https://github.com/scrapy/scrapy/pull/5246/commits/67ce305046398d08d626436c0d23376309cc4d67